InnovativeCars/force-app/main/default/classes/Constants.cls

```diff
@@ -7,11 +7,23 @@ public class Constants {
     public static final String PICKLIST_TEMPERATURE = 'Temperature';
     public static final String PICKLIST_ROTATION_SPEED = 'Rotation speed';
     public static final String PICKLIST_CLOSED = 'Closed';
+    public static final String PICKLIST_FAILED = 'Failed';
     public static final String FULL_SHOWROOM_ERROR = 'Cannot add a new car because the showroom is full!';
     public static final String ALREADY_AVAILABLE_ERROR = 'The car offer is already available at the showroom';
     public static final String NOT_IN_SHOWROOM_ERROR = 'The car is not in selected showroom';
     public static final String THERE_ALREADY_IS_WAREHOUSE_ERROR = 'Account of this showroom already has a warehouse: ';
     public static final String CANNOT_CLOSE_ERROR = 'Cannot close because there are cars left in the showroom';
     public static final String LESS_THAN_ZERO_ERROR = 'Number of car spaces cannot be less then 0!';
     public static final String LESS_THAN_CARS_AVAILABLE_ERROR = 'Number of car spaces cannot be less then number of cars available at the showroom!';
+    public static final String TELEMETRIES_DELETED_MESSAGE = 'Telemetries older than 90 days deleted successfully';
+    public static final String EMAIL_SEND_SUCCESSFULLY_MESSAGE = 'The email was sent successfully.';
+    public static final String EMAIL_SEND_FAILED_MESSAGE = 'The email failed to send: ';
+    public static final String NEW_OFFERS_EMAIL_ADDRESS = 'noviv37645@0ranges.com';
+    public static final String NEW_OFFERS_EMAIL_SUBJECT = 'New offers added!';
+    public static final String NEW_OFFERS_EMAIL_BODY = 'New offers have been added to showrooms. Number of offers: ';
+    public static final String CASE_SUBJECT = 'Renovation Case ';
+    public static final String CASE_ORIGIN = 'Web';
```

Comment:
the name is not meaningfull. the same as case subject. please consider something like CASE_ORIGIN_WEB (similarly with case subject)